libethereum/Account.cpp

```diff
@@ -118,35 +99,36 @@ AccountMap dev::eth::jsonToAccountMap(std::string const& _json, u256 const& _def
 
 	js::mValue val;
 	json_spirit::read_string_or_throw(_json, val);
-	js::mObject o = val.get_obj();
-	validateAccountMapObj(o);
-	for (auto const& account: o)
-	{
-		Address a(fromHex(account.first));
+
+    for (auto const& account : val.get_obj())
+    {
+        Address a(fromHex(account.first));
 		// FIXME: Do not copy every account object.
-		auto o = account.second.get_obj();
+        auto const& accountMapJson = account.second.get_obj();
```

Comment:
Maybe just `accountJson` or `accountObject`, because `acountMap` sounds like a map of accounts, but as I understand it refers to single account

Reply:
Map is because it indicates which fields are set and which are not. there is a class for that: AccountMap